Sources/MongoSwift/MongoCollection.swift

```diff
@@ -103,6 +122,41 @@ public struct MongoCollection<T: Codable> {
         return self._client.operationExecutor.execute(operation, client: self._client, session: session)
     }
 
+    /**
+     * Renames this collection with the specified options.
+     *
+     * - Parameters:
+     *   - to: A `String`, the new name for the collection
+     *   - dropTarget: A `Bool`, indicating whether the target collection should be dropped prior to renaming it.
```

Comment:
This option means to drop the destination collection before renaming the current one to it, ensuring the command will work even if the new collection name is already taken by an existing collection. The current docstring is a little unclear in how it expresses that, could it be updated to be more specific?